.storybook/decorators/withVModel.ts

```diff
@@ -13,9 +14,11 @@ export const withVModel = (story, { id, args }) => {
     name: 'DWithVModel',
     inheritAttrs: false,
     setup() {
+      const modelValueFallback = ref(null)
       const modelValue = computed({
-        get: ()=>(args.modelValue),
+        get: ()=>(modelValueFallback.value || args.modelValue),
```

Comment:
on Docs page Storybook update args only for "Basic" stories. This fallback allows to update the v-model for other stories